Doc/using/mac.rst

```diff
@@ -17,7 +17,8 @@ the IDE and the Package Manager that are worth pointing out.
 Getting and Installing MacPython
 ================================
 
-macOS since version 10.8 comes with Python 2.7 pre-installed by Apple.  If you wish, you
+macOS since version 10.8 comes with Python 2.7 pre-installed by Apple.
+And Python 2.7 has been removed since version 12.3, but Python 3 remains. If you wish, you
```

Comment:
That's also not entirely correct.  Python 3 is only available if you install Apple's developer tools.

Reply:
Yes, you are right, let me update it.